privval/dash_core_signer_client.go

```diff
@@ -225,7 +225,7 @@ func (sc *DashCoreSignerClient) GetProTxHash() (crypto.ProTxHash, error) {
 // SignVote requests a remote signer to sign a vote
 func (sc *DashCoreSignerClient) SignVote(
 	chainID string, quorumType btcjson.LLMQType, quorumHash crypto.QuorumHash,
-	protoVote *tmproto.Vote, logger log.Logger) error {
+	protoVote *tmproto.Vote, stateID tmproto.StateID, logger log.Logger) error {
```

Comment:
I wouldn't use the proto of the stateId. For the Vote I guess it makes sense because we are signing a vote message, but the state is no longer a message.

Reply:
Twisted logic :D